src/main/java/hillel/doctorRest/clinic/schedule/dto/ScheduleDtoConverter.java

```diff
@@ -11,6 +11,7 @@
     @Mappings({@Mapping(target = "id", ignore = true),
             @Mapping(target = "doctorId", ignore = true),
             @Mapping(target = "visitDate", ignore = true),
-            @Mapping(target = "hour", ignore = true)})
+            @Mapping(target = "hour", ignore = true),
```

Comment:
This is a misconception of the whole idea around mappers :(.

From one side, if you ignore all the fields, why do you even need this mapper?
From another side, why do you ignore all the fields if you set them by hand in service?